crates/bitwarden-core/Cargo.toml

```diff
@@ -76,8 +76,10 @@ rustls = { version = "0.23.19", default-features = false }
 rustls-platform-verifier = "0.6.0"
 
 [dev-dependencies]
-rand_chacha = "0.3.1"
+mockall = { version = ">=0.13.1, <0.15" }
+rand_chacha = ">=0.3.1, <0.4.0"
 tokio = { workspace = true, features = ["rt"] }
+wiremock = { workspace = true }
```

Comment:
We generally do not use wiremock any longer and instead use the `ApiClient::new_mocked`. There are a couple of examples of this being used.